.travis.yml

```diff
@@ -23,6 +23,7 @@ script:
   - FORCE_COLOR=1 npx react-jsx-i18n extract --ext jsx indico/web/client/ indico/modules/ > /dev/null
   - isort -rc --diff --check-only indico/
   - python bin/maintenance/update_backrefs.py --ci
+  - python bin/maintenance/update_header.py indico
```

Comment:
`--ci` even though it doesn't really matter since we don't use travis anymore on master